HLTrigger/special/plugins/HLTPPSJetComparisonFilter.cc

```diff
@@ -69,7 +69,10 @@ void HLTPPSJetComparisonFilter::fillDescriptions(edm::ConfigurationDescriptions
   desc.add<edm::InputTag>("forwardProtonInputTag", edm::InputTag("ctppsProtons", "singleRP"))
       ->setComment("input tag of the forward proton collection");
 
-  desc.add<std::string>("lhcInfoLabel", std::string(""))->setComment("label used for LHCInfo");
+  desc.add<std::string>("lhcInfoLabel", "")->setComment("label used for LHCInfo");
+  desc.add<std::string>("lhcInfoPerLSLabel", "")->setComment("label of the LHCInfoPerLS record");
+  desc.add<std::string>("lhcInfoPerFillLabel", "")->setComment("label of the LHCInfoPerFill record");
+  desc.add<bool>("useNewLHCInfo", false)->setComment("flag whether to use new LHCInfoPer* records or old LHCInfo");
```

Comment:
as this module is not already in use, why not setting the default to `true` (as meant for production in 2024)?

Reply:
What if this module would be used in `DirectSimulation` or with GT from `Run 2`?

Reply:
is the `DirectSimulation` exercised anywhere?

Reply:
This code will never be used with `Run 2` data, right?

Reply:
I don't see how it can be used for Run2 data, given that the HLT for past data has already run.
The only remote use case I can see is if we want to use this filter in Run2  HLT simulation, but it makes little sense to me since the trigger using it was not active during data-taking.